main.c

```diff
@@ -1578,13 +1622,12 @@ static void registry_get(void *data, struct wl_registry *r, uint32_t id,
 			 const char *i, uint32_t version)
 {
 	if (strcmp(i, "wl_compositor") == 0) {
-		term.cp = wl_registry_bind(r, id, &wl_compositor_interface, 1);
+		term.cp = wl_registry_bind(r, id, &wl_compositor_interface, 3);
```

Comment:
Hmm, we might still want to support older compositors, will need to look into it.